### PR TITLE
Third party review UI

### PR DESCRIPTION
**Pull request by Macroz**
Implement the third party review request flow
- use komponentit/autocomplete but wrap inside own component

### CODE REVIEW

**Comment by Macroz** [comment]
Implements UI for #458

File: resources/translations/en-GB.edn [hunk 1]
```diff
@@ -95,6 +95,7 @@
                           :third-party-review "Reviewed by a third party"
                           :unknown "Unknown"
                           :withdraw "Withdrawn for modifying"}
+                 :request-review "Request review by a third party"
```
Comment (tjalkane) [inline resources/translations/en-GB.edn]: Why is this needed? What about `:t.actions/review-request`?

Comment (Macroz) [reply resources/translations/en-GB.edn]: If that's not used elsewhere then it could be used instead and this one removed.

File: src/clj/rems/css/styles.clj [hunk 2]
```diff
@@ -347,6 +347,56 @@
...
+             ;;:border-radius (u/px 4)
+             ;;:border [[(u/px 1) :solid "#111"]]
+
+             ;; from Bootstrap .form-control
+             :display :block
+             :width (u/percent 100)
+             :padding [[(u/rem 0.375) (u/rem 0.75)]]
+             :font-size (u/rem 1)
+             :line-height 1.5
+             :color "#495057"
```
Comment (tjalkane) [inline src/clj/rems/css/styles.clj]: These random hardcoded colours should conform to existing theme and probably be configurable as well

File: src/cljs/rems/application.cljs [hunk 3]
```diff
@@ -438,6 +439,142 @@
...
+                 :recipients (map :userid reviewers)}
+        :handler (fn [resp]
+                   (rf/dispatch [::send-3rd-party-review-request-success true]))}))
+
+(rf/reg-event-fx
+ ::send-3rd-party-review-request
+ (fn [{:keys [db]} [_ reviewers comment]]
+   (let [application-id (get-in db [:application :application :id])
+         round (get-in db [:application :application :curround])
+         user (get-in db [:identity :user])]
```
Comment (opqdonut) [inline src/cljs/rems/application.cljs]: is it weird that some params come from the event and some come from the db?

File: src/cljs/rems/application.cljs [hunk 4]
```diff
@@ -438,6 +439,142 @@
...
+        potential-3rd-party-reviewers @(rf/subscribe [::potential-3rd-party-reviewers])
+        review-comment @(rf/subscribe [::review-comment])]
+    [:div.modal.fade {:id "review-request-modal" :role "dialog" :aria-labelledby "confirmModalLabel" :aria-hidden "true"}
+     [:div.modal-dialog {:role "document"}
+      [:div.modal-content
+       [:div
+        [:div.modal-header
+         [:h5#confirmModalLabel.modal-title (text :t.applications/request-review)]
+         [:button.close {:type "button" :data-dismiss "modal" :aria-label (text :t.actions/cancel)}
+          [:span {:aria-hidden "true"} "\u00D7"]]]
```
Comment (opqdonut) [inline src/cljs/rems/application.cljs]: [minor] should this be an atom?

Comment (Macroz) [reply src/cljs/rems/application.cljs]: Lets do that if/when we get another modal.

File: src/cljs/rems/spa.cljs [hunk 5]
```diff
@@ -88,6 +88,8 @@
 (secretary/defroute "/application/:id" {id :id}
   (rf/dispatch [:rems.application/zero-state])
   (rf/dispatch [:rems.application/start-fetch-application id])
+  (rf/dispatch [:rems.application/start-fetch-potential-3rd-party-reviewers])
+  (rf/dispatch [:rems.application/send-3rd-party-review-request-success false])
```
Comment (opqdonut) [inline src/cljs/rems/spa.cljs]: should this be in zero-state?
